src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/cas1/Cas1AssessmentEmailService.kt

```diff
@@ -50,6 +50,25 @@ class Cas1AssessmentEmailService(
     }
   }
 
+  fun appealedAssessmentAllocated(allocatedUser: UserEntity, assessmentId: UUID, crn: String) {
```

Comment:
if is it worth updating an integration test to make sure this is sent in this circumstance?

Reply:
Oh good call - I also realised I had the logic in the reallocation method, not the assessment creation method, so have fixed that now 😅